tests/dogtag/pytest-ansible/installation/main.retry

```diff
@@ -0,0 +1 @@
+10.16.70.63
```

Comment:
Could we not check in `.retry` files? And perhaps add `*.retry` to `.gitignore`.

Reply:
Thanks!

Reply:
I have removed it but yes it's good to ignore such things in .gitignore.